src/components/html/EcAdminSettingsForm.html

```diff
@@ -85,16 +100,15 @@
               component,
               data
             }) in parseAdminSettingsField(config)"
-            v-if="component"
+            v-if="component && !checkComplexSchema(schema)"
             :key="name"
-            :label="schema.title"
+            :label="header !== schema.title ? schema.title || field: ''"
             :label-col="{ span: 8 }"
             :wrapper-col="{ span: 16 }"
           >
-            <template v-if="schema.description" #extra>
+            <template v-if="schema.description && !checkComplexSchema(schema)" #extra>
```

Comment:
Vale o mesmo que comentei sobre o label...

```suggestion
            <template v-if="schema.description" #extra>
```